plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -124,6 +124,11 @@ class RayAWSConf(RayConf):
     # (Cluster will remain provisioned and can be started with "ray up cluster.yaml").
     stop_cluster: bool = MISSING
 
+    # if your code contains more than one modules/files, you need to sync your code to the cluster.
+    # The code will be synced to a tmp dir on the cluster.
+    # Leave this empty if you don't need to sync code.
+    code_dir: str = MISSING
```

Comment:
1. What if there are multiple directories to synchronize?
2. What if there are files to exclude?